ensembl/htdocs/info/genome/variation/species/populations.html

```diff
@@ -251,7 +270,187 @@ <h3>Populations from  the <a href="https://www.internationalgenome.org/data-port
 
 
   <div style="margin-bottom:35px"></div>
-</div>
+
+      <h3>Populations from  the <a href="https://evs.gs.washington.edu/EVS/" target="_blank" style="text-decoration:none">NHLBI Exome Sequencing Project</a></h3>
+      <table id="nhlbi_exome_sequencing_project" class="ss" style="margin-bottom:5px">
+  
+  <tr>
+    <th style="width:200px">Name</th>
+    <th style="width:36px">Size</th>
+    <th>Description</th>
+  </tr>
+
+  <tr>
+    <td>ESP6500:<b>AA</b></td>
+    <td style="text-align:right">-</td>
+    <td>African American</td>
+  </tr>
+  <tr class="bg2">
+    <td>ESP6500:<b>EA</b></td>
+    <td style="text-align:right">-</td>
+    <td>European American</td>
+  </tr>
+</table>
+
+
+  <div style="margin-bottom:35px"></div>
+
+      <h3>Populations from <a href="https://www.uk10k.org/" target="_blank" style="text-decoration:none">UK10K</a></h3>
+      <table id="uk10k" class="ss" style="margin-bottom:5px">
+  
+  <tr>
+    <th style="width:200px">Name</th>
+    <th style="width:36px">Size</th>
+    <th>Description</th>
+  </tr>
+
+  <tr>
+    <td>UK10K:<b>ALSPAC</b></td>
+    <td style="text-align:right">-</td>
+    <td>ALSPAC cohort</td>
+  </tr>
+  <tr class="bg2">
+    <td>UK10K:<b>TWINSUK</b></td>
+    <td style="text-align:right">-</td>
+    <td>TWINSUK cohort excluding 67 samples where a monozygotic or dyzygotic twin was included in the release</td>
+  </tr>
```

Comment:
I don't know where we are getting these descriptions from, but it would be nice to improve the description of these databases. For instance:

```suggestion
  <tr>
    <td>UK10K:<b>ALSPAC</b></td>
    <td style="text-align:right">-</td>
    <td>Avon Longitudinal Study of Parents and Children (ALSPAC) cohort</td>
  </tr>
  <tr class="bg2">
    <td>UK10K:<b>TWINSUK</b></td>
    <td style="text-align:right">-</td>
    <td>UK twin registry of identical and non-identical twins (16 to 85 years old), excluding 67 samples where a monozygotic or dyzygotic twin was included in the release</td>
  </tr>
```

Reply:
This is coming from the vcf config file